# goal
bool pregrasp
bool away
string object_name
#geometry_msgs/Pose[] target_positions
#int32[] positions_ids
---
# result
bool success
bool failed
bool repeat
bool timeout
bool collision
duration time_elapsed
#int32 reached_position
#geometry_msgs/Pose reached_position
#bool pregrasps_not_suitable
---
# feedback
bool starting
bool planning
bool executing
bool reset